terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -105,12 +103,22 @@ public void test01_03_redirectToExternalLink() {
         // screen capture
         screenCapture.save(driver);
 
+        ApServerName apServerName = webDriverOperations.getApServerName();
+        String apServerVersion = webDriverOperations.getApServerVersion();

```

Comment:
Comment: `// Unlike other application servers, WebSphere Liberty Profile 16 & WebSphere traditional 9 wraps an unexpected exception of ServletException.`

Leave Websphere versions yet...